app/Actions/Diagnostics/Checks/BasicPermissionCheck.php

```diff
@@ -156,15 +187,20 @@ private function checkDirectoryPermissionsRecursively(string $path, array &$erro
 			if ($expectedPerm !== $actualPerm) {
 				$this->numPermissionIssues++;
 				if ($this->numPermissionIssues <= self::MAX_ISSUE_REPORTS_PER_TYPE) {
-					$errors[] = sprintf('Warning: %s has permissions %04o, but should have %04o at least and %04o at most', $path, $actualPerm, FixPermissions::MIN_DIRECTORY_PERMS, FixPermissions::MAX_DIRECTORY_PERMS);
+					$errors[] = sprintf(
+						'Warning: %s has permissions %04o, but should have %04o',
+						$path,
+						$actualPerm,
+						$expectedPerm
+					);
 				}
 			}
 
 			if (!is_writable($path) || !is_readable($path)) {
 				$this->numAccessIssues++;
 				if ($this->numAccessIssues <= self::MAX_ISSUE_REPORTS_PER_TYPE) {
 					$problem = match (true) {
-						(!is_writable($path) && !is_readable($path)) => 'readable and writable',
+						(!is_writable($path) && !is_readable($path)) => 'readable nor writable',
```

Comment:
```suggestion
						(!is_writable($path) && !is_readable($path)) => 'readable or writable',
```

Reply:
nah, it that case it is both, if you use `or` the semantic is lost.

Reply:
I'm not a native speaker so I don't want to sound all-authoritative here, but my understanding has always been that "not readable nor writable" simply isn't a correct English; it should be either "not readable or writable" (which, I agree, seems a little ambiguous) or "neither readable nor writable".

Reply:
neither is indeed better.

Reply:
Fixed in https://github.com/LycheeOrg/Lychee/pull/1351/commits/8f9e35a1d193013267d995704e996a0c9339ea2d